frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -468,6 +469,25 @@ export const createOrUpdateResources = async (
         generatedImageStreamName,
       ),
     );
+  } else if (!_.isEmpty(pipeline.data) && !dryRun) {
+    const updatedPipeline = await updatePipelineForImportFlow(
+      pipeline,
+      formData.name,
+      formData.project.name,
+      formData.git.url,
+      formData.git.ref,
+      formData.git.dir,
+      formData.docker.dockerfilePath,
+    );
+    if (_.has(updatedPipeline.metadata.labels, 'pipeline.openshift.io/runtime')) {
```

Comment:
@rottencandy what was the reason for adding this check for `pipeline.openshift.io/runtime` label? I am asking because this wont be present in the pipeline that is associated with an app created using import from dockerfile and thus if the this app is edited, pipeline-run will not be started automatically.

Reply:
I added this check to make sure that the pipeline is still associated to the application, realized now that I should be checking for the `app.kubernetes.io/instance` label instead. Updated.

Reply:
Yes. Thanks!